rest/importtest/import_test.go

```diff
@@ -2467,3 +2467,31 @@ func TestImportUpdateExpiry(t *testing.T) {
 		})
 	}
 }
+
+func TestDoNotWriteBodyBackOnImport(t *testing.T) {
+	base.SkipImportTestsIfNotEnabled(t)
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyHTTP, base.KeyCRUD, base.KeyImport)
+
+	rt := rest.NewRestTester(t, &rest.RestTesterConfig{
+		AutoImport: base.Ptr(true),
+	})
+	defer rt.Close()
+
+	docID := t.Name()
+
+	collection, _ := rt.GetSingleTestDatabaseCollectionWithUser()
+	ds := collection.GetCollectionDatastore()
+
+	specBody := []byte(`{"test":"<>"}`) // use a special character that is currently escaped to ensure it is not modified by the import process
```

Comment:
I think this import codepath is updating the `doc_writes_bytes` stat via:

```go
docBytes := 0   // Track size of document written, for write stats

// ...

// if isImport is true, we don't want to update the document body, only the xattrs
if !isImport && len(rawDocBody) > 0 {
	updatedDoc.Doc = rawDocBody
	docBytes = len(updatedDoc.Doc)

// ...

db.dbStats().Database().DocWritesBytes.Add(int64(docBytes))
```

which might be a useful assertion that continues working for if/when we fix the `<>` bug